src/orange/detail/OrientedBoundingZone.hh

```diff
@@ -18,21 +18,34 @@ namespace celeritas
 namespace detail
 {
 //---------------------------------------------------------------------------//
-/*!
- * Oriented bounding zone for saftey distance calculations.
+/*! Oriented bounding zone for safety distance calculations.
+ *
+ * Here, the oriented bounding zone (OBZ) is defined by inner and outer
+ * bounding boxes transformed by the same transformation --- that of the volume
+ * they correspond to. The OBZ bounding boxes are stored on the
+ * OrientedBoundingZoneRecord as:
+ *
+ * 1. a vector of half-widths,
+ *
+ * 2. an additional *translation* object for each bounding box specifying how
+ * the center of each bounding box is offset from the center of the OBZ
+ * coordinate system.
  *
- * Here, the bounding zone is defined by an inner and outer bounding box
- * (specified via half-widths) transformed by the same transformation;
- * pressumably the transformation of the volume they correspond to. The
- * safety distance can be approximated as the minimum distance to the inner or
- * outer box. For points lying between the inner and outer boxes, the safety
+ * Consequently, points in the unit's corrdinate system must be first
+ * transformed by <transform_id> into the OBZ coordinate system (resulting in
+ * "trans_pos"), then offset, i.e. translated, by <inner_offset_id> or
+ * <outer_offset_id> into the inner or outer bbox coordinate system (resulting
+ * in "offset_pos"). It is noted that these offset positions are always
+ * automatically reflected into the first quadrant.
+ *
+ * The safety distance can be approximated as the minimum distance to the inner
+ * or outer box. For points lying between the inner and outer boxes, the safety
  * distance is zero.
  */
 class OrientedBoundingZone
 {
   public:
-    //!@{
-    //! \name Type aliases
+    //!@{ ! \name Type aliases
```

Comment:
Something messed up the comment here, also strangely deleting a line in the class comment above.